rllib/tests/test_env_with_subprocess.py

```diff
@@ -28,7 +28,7 @@ def leaked_processes():
     _, tmp4 = tempfile.mkstemp("test_env_with_subprocess")
     register_env("subproc", lambda c: EnvWithSubprocess(c))
 
-    ray.init()
+    ray.init(local_mode=True)
```

Comment:
Revert this? Local mode won't be testing subprocesses correctly.

Reply:
Perfect, thanks!